wazimap_ng/points/dataloader.py

```diff
@@ -12,14 +12,14 @@ def loaddata(category, iterable, row_number):
     for idx, row in enumerate(iterable):
 
         if "longitude" not in row:
-            logs.append([row_number+idx, "longitude", "Missing Header Longitude"])
+            logs.append([row_number+idx+1, "longitude", "Missing Header Longitude"])
```

Comment:
what about instead of repeating the `row_number+idx+1` in 6 lines moving this into a variable? 
Then you can have a nice name and the +1 does not lead to confusion.

Reply:
👍